azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -79,7 +79,7 @@ def enable(self):
         def disable(self):
             self._enabled = False
 
-        def _invoke_cgroup_operation(self, operation, error_message):
+        def _invoke_cgroup_operation(self, operation, error_message, on_error=None):
             """
             Ensures the given operation is invoked only if cgroups are enabled and traps any errors on the operation.
             """
```

Comment:
Can you update the comment explaining what `on_error` does and why it's there?

Reply:
doesn't the name imply a callback to execute when an error occurs? :)